hack/make-rules/verify.sh

```diff
@@ -21,15 +21,10 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/cluster/lib/util.sh"
 
-if [ -n "${VERBOSE}" ]; then
```

Comment:
The VERBOSE=1 flag is still specified in "hack/jenkins/verify.sh" and "hack/jenkins/verify-dockerized.sh"
Perhaps, it should be removed from there, as part of removing this..